common/close_tx.c

```diff
@@ -46,6 +49,9 @@ struct bitcoin_tx *create_close_tx(const tal_t *ctx,
 		script = tal_dup_talarr(tx, u8, our_script);
 		/* One output is to us. */
 		bitcoin_tx_add_output(tx, script, NULL, to_us);
+		if (local_wallet_index && local_wallet_ext_key)
```

Comment:
they can only be both set or both unset, no?  if that's the case, maybe assert their nullity is the same - `(local_wallet_index == NULL) == (local_wallet_ext_key == NULL)`.

Reply:
fixed